components/automate-chef-io/content/docs/configuration.md

```diff
@@ -277,20 +277,25 @@ Uncomment and change settings as needed, and then run `chef-automate config patc
 [elasticsearch.v1.sys.runtime]
 # max_locked_memory = "unlimited"
 # es_java_opts = ""
-# NOTE: heapsize should be set to 50% of available RAM up to 32g
-# See: https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html
-# heapsize = "1g"
+# NOTE: see https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html
+# for important guidance regarding the configuration of the heap size setting
```

Comment:
```suggestion
# NOTE: [Elasticsearch Heap Sizing](https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html)
# contains important guidance regarding the configuration of the heap size setting
```

wait, idk our standard for having links in the config comments 🤷

Reply:
I like the wording change, but yeah, is using `[]()` a thing here since it won't actually be rendered as a link?

Reply:
You would need a shortcode for the note to render correctly, in which case you're right that the link wouldn't work correctly. In this case, # NOTE will probably render as a H1, which is  not what we want.

I think we're OK dropping `# NOTE:` and using the rest of the sentence. Bonus points if you can get it into the active voice.

Reply:
@kagarmoe My point was that this portion is all already inside `<pre><code>` in the rendered documentation, so is there a purpose to keeping `[]()` or removing `NOTE:` ? I might not understand all the places where this content ends up.

Reply:
Ah, I didn't see that it was inside of a code fence. The backstory is that supplying the link was a compromise & was included here for people who would just copy-paste the example and not read the whole doc. You can keep it or remove it, I don't have a strong opinion either way. I'm happy to see us update the heapsize.

Reply:
The code fences/<pre><code> mean that it renders exactly as we see it. @susanev may be trying to remove `see`? However, the idiom is accepted in major house style guides, meaning Microsoft & Google. This is an example of a case where trying to avoid it is unhelpful. From the docs perspective, you can reject this change.

Reply:
oh no i was just trying to get link text in there but no worries if its not possible

Reply:
Sadly, it won't work because of the code fences.